eland/ml/transformers/xgboost.py

```diff
@@ -155,17 +153,16 @@ def determine_target_type(self) -> str:
     def is_objective_supported(self) -> bool:
         return False
 
-    def check_model_booster(self) -> None:
+    @staticmethod
+    def check_model_booster(booster: Optional[str]) -> None:
```

Comment:
Since we're already calling this function in two places, thoughts on pulling the `booster` string grabbing logic into this function and passing the model as a parameter?